### PR TITLE
Update "Test the Implementation" With iOS Push Notification Section

### PR DESCRIPTION
**Pull request by ConnorLand**
**This PR contains the following changes:**

* Corrected, step-by-step text for the Introduction section
* A new section (Using APNS Instead of FCM for IOS Devices) which a Mendix employee suggested we include
* An anchor link to the [Downloading the Google Services Config Files](setting-up-google-firebase-cloud-messaging-server#downloading-the-google-services-config-files) section of _Set Up the Google Firebase Cloud Messaging Server_

**Questions**

* Should "iOS" be "IOS" as I typed it in a heading? 
* Is section 1.1 best as a sub-section, or would it be better as blue info text?

### CODE REVIEW

**Comment by MariaShaposhnikova** [comment]
Connor, 
Concerning your questions:
* Should "iOS" be "IOS" as I typed it in a heading?
  I did some research and looked up how we use in our titles and came to a conclusion that we should spell it "iOS" in titles.
* Is section 1.1 best as a sub-section, or would it be better as blue info text?
  Now that I see it in the preview, I think it looks OK as a sub-section as well.

**Comment by ConnorLand** [comment]
> Connor,
> Concerning your questions:
> 
> * Should "iOS" be "IOS" as I typed it in a heading?
>   I did some research and looked up how we use in our titles and came to a conclusion that we should spell it "iOS" in titles.
> * Is section 1.1 best as a sub-section, or would it be better as blue info text?
>   Now that I see it in the preview, I think it looks OK as a sub-section as well.

Ok thanks for your changes @MariaShaposhnikova

File: content/howto/mobile/testing-the-implementation.md [hunk 1]
```diff
@@ -5,13 +5,31 @@ menu_order: 60
 tags: ["mobile", "push notification"]
 ---
 
-## 1 Introuction
+## 1 Introduction
 
-Once you finish implementing the steps described in [How to Implement Push Notifications](implementation-guide), you need to test whether everything works correctly. As a prerequisite, you will need to build (new) versions of your mobile application. You can refer to the [hybrid app package documentation](https://github.com/mendix/hybrid-app-template/) for instructions. Please note that, during this process, you will need to put the *google-services.json* and *GoogleService-Info.plist* files in the config folder.
+Once you finish implementing the steps described in [How to Implement Push Notifications](implementation-guide), you need to test whether everything works correctly. As a prerequisite, you will need to build new versions of your mobile application. To build an app which supports push notifications, follow these instructions:
```
Comment (MariaShaposhnikova) [inline content/howto/mobile/testing-the-implementation.md]: "As a prerequisite, you will need to build new versions of your mobile application. To build an app which supports push notifications, follow these instructions:"
I am a bit confused with this one. Are steps below prerequisites as well? Does "an app which supports push notifications" help you "build new versions of your mobile application"?

Comment (ConnorLand) [reply content/howto/mobile/testing-the-implementation.md]: I've committed a change which may help clear this paragraph up.

**Comment by ConnorLand** [comment]
I made some changes to this doc per Maria's guidance on slack. In a subsequent PR, I plan to add Read More sections to all the docs in this Hybrid series.

**Comment by MariaShaposhnikova** [comment]
As a follow-up of our discussion with @ConnorLand, we agreed on the following changes:
1. To add the Prerequisites section.
2. To make minor changes to the last section to make it look more complete and correlate with the title.

**Pull request merged**